.github/workflows/release-static-production.yml

```diff
@@ -200,4 +213,8 @@ jobs:
 
       - name: Deploying
         run: |
-          npx gulp staticDeploy
+          if ${{ github.ref_name }} != "production-amp-dev" ; then
```

Comment:
Do I understand this right, if we're not merging to production-amp-dev, then we should should deploy to prod? 

Sidenote: I find the flag `stage` confusing. Does this mean `staging`? Maybe `prod` would make this more clear.

Reply:
ah, flipped the logic in gulp, but missed this. I felt like the expected default of `deploy` is to prod, so adding a flag to modify that to the less expected staging was the reasoning. I will change it to deploy to staging by default and to prod only when flagged as `--prod`

Reply:
both is fine as long as the logic is right :-)